WooCommerce/Classes/ViewRelated/Inbox/InboxNoteRow.swift

```diff
@@ -7,6 +7,9 @@ struct InboxNoteRow: View {
 
     // Tracks the scale of the view due to accessibility changes.
     @ScaledMetric private var scale: CGFloat = 1
+    @State private var displayedURL: URL?
+    @State private var showWebView: Bool = false
```

Comment:
nit: `show*` sounds like a function, how about `shows*`? or is this a SwiftUI pattern?
```suggestion
    @State private var showsWebView: Bool = false
```